nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -546,6 +546,28 @@ def quantize_output(self, output, wrapper, **kwargs):
         out = self._dequantize(module, out)
         return out
 
+    def load_calibration_config(self, calibration_config):
+        modules_to_compress = self.get_modules_to_compress()
+        for layer, config in modules_to_compress:
+            name, module = layer.name, layer.module
+            if name not in calibration_config:
+                if hasattr(module, 'weight_bits') or hasattr(module, 'output_bits'):
```

Comment:
Because `input` quantize is always combined with `weight` quantize. Add `intput_bits` here is more clear and may avoid some corner cases. Have added.

Reply:
got it, what will happen if the user only sets 'input_bits' in config?

Reply:
No thing will happen during simulation process. However, if we deploy it on inference framework, computation of this op will be converted to 32bit and warning will be raised since input tensor and weight tensor have different bit-width.